deploy/docs/additional_prometheus_configuration.md

```diff
@@ -158,17 +190,30 @@ kind: Secret
 
 ### Update the prometheus-overrides.yaml file to forward the metrics to Fluentd.
 
-The `prometheus-overrides.yaml` file controls what metrics get forwarded on to Sumo Logic. To send custom metrics to Sumo Logic you need to update the `prometheus-overrides.yaml` file to include a rule to forward on your custom metrics. Make sure you include the same tag you created in your Fluentd configmap in the previous step. Here is an example addition to the `prometheus-overrides.yaml` file that will forward metrics to Sumo:
+The configuration file controls what metrics get forwarded on to Sumo Logic. To send custom metrics to Sumo Logic you need to update it to include a rule to forward on your custom metrics. Make sure you include the same tag you created in your Fluentd configmap in the previous step. Here is an example addition to the configuration file that will forward metrics to Sumo:
 
-```
+```yaml
 - url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.YOUR_TAG
       writeRelabelConfigs:
       - action: keep
         regex: <YOUR_CUSTOM_MATCHER>
         sourceLabels: [__name__]
 ```
 
-Replace `YOUR_TAG` with a tag to identify these metrics. After adding this to the `yaml`, go ahead and run a `helm upgrade prometheus-operator stable/prometheus-operator -f prometheus-overrides.yaml` to upgrade your `prometheus-operator`.
+According to our example, below config could be useful:
+
+```yaml
+- url: http://collection-sumologic.sumologic.svc.cluster.local:9888/prometheus.metrics.YOUR_TAG
+      writeRelabelConfigs:
+      - action: keep
+        regex: '^example-metrics$'
+        sourceLabels: [service]
+```
+
+Replace `YOUR_TAG` with a tag to identify these metrics. After adding this to the `yaml`, go ahead and upgrade your sumologic or prometheus operator installation, depending on used method:
+
+* `helm upgrade collection sumologic/sumologic -f <path to values.yaml>` to upgrade sumologic collection
```

Comment:
it should use `--reuse-values`

Reply:
change "depending on used method" to "depending on method used"